### PR TITLE
Table theme + stories

### PR DESCRIPTION
**Pull request by ShimiSun**
#### What does this PR do?
* Organizing Table stories
* Adds theme docs to table
#### Where should the reviewer start?
doc.js files
#### What testing has been done on this PR?
Custom Table on storybook
#### How should this be manually tested?
Custom Table on storybook + Table storybook
#### What are the relevant issues?
#2690 
#### Do the grommet docs need to be updated?
done
#### Should this PR be mentioned in the release notes?
nope
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

File: src/js/components/TableCell/doc.js [hunk 1]
```diff
@@ -42,3 +42,91 @@ export const doc = TableCell => {
 
   return DocumentedTableCell;
 };
+
+export const themeDoc = {
+  'table.body.align': {
```
Comment (oorestisime) [inline src/js/components/TableCell/doc.js]: they should go in tableBody no?

File: src/js/components/TableCell/doc.js [hunk 2]
```diff
@@ -42,3 +42,91 @@ export const doc = TableCell => {
...
+    description: 'Any additional style for Table body.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'table.body.pad': {
+    description: 'The padding of the body.',
+    type: 'string | object',
+    defaultValue: "{ horizontal: 'small', vertical: 'xsmall' }",
+  },
+  'table.footer.align': {
```
Comment (oorestisime) [inline src/js/components/TableCell/doc.js]: this should go in tableFooter ?

File: src/js/components/TableCell/doc.js [hunk 3]
```diff
@@ -42,3 +42,91 @@ export const doc = TableCell => {
...
+    description: 'The padding of the footer.',
+    type: 'string | object',
+    defaultValue: "{ horizontal: 'small', vertical: 'xsmall' }",
+  },
+  'table.footer.verticalAlign': {
+    description: 'How to align the content vertically.',
+    type: 'string',
+    defaultValue: 'top',
+  },
+  'table.header.align': {
```
Comment (oorestisime) [inline src/js/components/TableCell/doc.js]: Following in header

**Comment by ShimiSun** [comment]
@oorestisime I had the same thought at the beginning, but I followed the current structure where these styles are explicitly defined in TableCell and used in StyledTableCell+StyledTable and not on the TableHeader/ TableFooter/ TableBody. https://github.com/grommet/grommet/blob/43a3c09b8ee449096ad6764497a27bc30167ef1d/src/js/components/TableCell/TableCell.js#L25

**Comment by oorestisime** [comment]
Yeah sorry i hadn't read the code before commenting! Makes much more sense now!

File: src/js/components/TableCell/doc.js (hunk 1 above)
Comment (ericsoderberghp) [reply src/js/components/TableCell/doc.js]: I can't see where this is being used in the code.

**Pull request merged**